src/components/NavMobile/NavMobile.js

```diff
@@ -0,0 +1,87 @@
+//Dependencies
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+//Components, assets, actions, styles etc..
+import Button from '../Button/Button';
+import { ReactComponent as HUOALogo } from '../../assets/huoa-logo.svg';
+import { ReactComponent as Hamburger } from '../../assets/hamburger.svg';
+import { ReactComponent as ButtonClose } from '../../assets/button-close.svg';
+import { ReactComponent as Carrot } from '../../assets/carrot.svg';
+import './NavMobile.scss';
+
+const NavMobile = () => {
+  const [ navState, setNavState ] = useState('init');
+  const [ activeDropdown, setActiveDropdown ] = useState(null);
+  const { routes } = useSelector((state) => state.router);
+
+  console.log(navState);
+
+  const mobileRoutes = routes.filter((item) => item.name !== 'Donate');
+  mobileRoutes.unshift({ name: 'Home', linkTo: '/' });
+
+  const renderDropdown = (items) => {
+    return items.map((item) => {
+      return (
+        <Link to={item.linkTo} key={item.linkTo}>
+          <li className="nav-mobile__sidebar--dropdown-item">{item.name}</li>
+        </Link>
+      );
+    });
+  };
+
+  const renderSideItems = (items) => {
+    return items.map((item) => {
+      const dropBool = activeDropdown === item.name;
+
+      return (
+        <React.Fragment key={item.linkTo}>
+          <Link to={item.children ? '#!' : item.linkTo} className="nav-mobile__sidebar--item">
+            <li>{item.name}</li>
+            {item.children && (
+              <Carrot
+                className={`nav-mobile__sidebar--carrot${dropBool ? '--active' : ''}`}
+                onClick={() => setActiveDropdown(dropBool ? null : item.name)}
+              />
+            )}
+          </Link>
+          <div className={`nav-mobile__sidebar--dropdown${dropBool ? '--active' : ''}`}>
+            {dropBool && renderDropdown(item.children)}
+          </div>
+        </React.Fragment>
+      );
+    });
+  };
+
+  return (
+    <React.Fragment>
+      <nav className="nav-mobile">
+        <div className="nav-mobile__header">
+          <HUOALogo className="nav-mobile__logo" height="90" width="90" />
```

Comment:
BAD PRACTICE

```suggestion
          <HUOALogo className="nav-mobile__logo" />
```

should be handled in the stylesheet